testing/unit_tests/test_HeMIS.py

```diff
@@ -40,9 +42,9 @@ def setup_function():
     "extensions": [".nii.gz"],
     "roi_params": {"suffix": "_seg-manual", "slice_filter_roi": None},
     "contrast_params": {"contrast_lst": ['T1w', 'T2w', 'T2star']
-    }}])
+                        }}])
 @pytest.mark.run(order=1)
-def test_HeMIS(loader_parameters, p=0.0001):
+def test_HeMIS(get_data_testing_test_files, loader_parameters, p=0.0001, ):
```

Comment:
trailing comma left after last function arg

Reply:
Great catch...! Python is being too tolerant from time to time (I am surprised pytest past lol), I had to hunt Tuple bug from time to time because of typos like this... Fixed.